proto/cosmos/accounts/vesting/tx.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+package cosmos.accounts.vesting;
+
+import "amino/amino.proto";
+import "google/protobuf/any.proto";
+import "cosmos/accounts/vesting/vesting.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/msg/v1/msg.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "cosmossdk.io/x/accounts/vesting/types";
+
+//-------------------------------------- INIT --------------------------------------
+
+// MsgInitVestingAccount defines a message that enables creating a vesting
+// account.
+message MsgInitVestingAccount {
+  option (amino.name) = "cosmos-sdk/MsgInitVestingAccount";
+
+  option (gogoproto.equal) = true;
+
+  // owner of the vesting account
+  string   owner                           = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  repeated cosmos.base.v1beta1.Coin amount = 2 [
+    (gogoproto.nullable)     = false,
+    (amino.dont_omitempty)   = true,
+    (amino.encoding)         = "legacy_coins",
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+
+  // end of vesting
+  google.protobuf.Timestamp end_time = 3
+      [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdtime) = true];
+  // start of vesting
+  google.protobuf.Timestamp start_time = 4
+      [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdtime) = true];
+}
+
+// MsgInitVestingAccountResponse defines the Msg/InitVestingAccount response type.
+message MsgInitVestingAccountResponse {}
+
+// MsgInitVestingAccount defines a message that enables creating a vesting
+// account.
+message MsgInitPeriodicVestingAccount {
+  option (amino.name) = "cosmos-sdk/MsgInitPeriodVestAccount";
+
+  option (gogoproto.equal) = false;
+
+  // owner of the vesting account
+  string owner = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // start of vesting
+  google.protobuf.Timestamp start_time = 2
+      [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdtime) = true];
+  repeated Period vesting_periods = 3 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+}
+
+// MsgInitVestingAccountResponse defines the Msg/InitPeriodicVestingAccount
+// response type.
+message MsgInitPeriodicVestingAccountResponse {}
+
+// MsgExecuteMessages defines a message that enable vesting account to execute moule messages
```

Comment:
```suggestion
// MsgExecuteMessages defines a message that enable vesting account to execute module messages
```

nit: I guess "module messages" refers to normal messages?

Reply:
yesssss